From 45c180b783044a27c630ea0c26b1dbef06bd1c5a Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Thu, 3 Aug 2017 15:23:54 -0700
Subject: [PATCH] TOOLS-1830 replace sdcadm experimental nfs-volumes with
 cloudapi-nfs-volumes/docker-nfs-volumes subcommands TOOLS-1829 add sdcadm
 experimental docker-nfs-volumes-automount

---
 tools/setup/setup.sh | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/tools/setup/setup.sh b/tools/setup/setup.sh
index f2f471a..f420625 100644
--- a/tools/setup/setup.sh
+++ b/tools/setup/setup.sh
@@ -317,9 +317,12 @@ upgrade_core_service_to_latest_branch_image "volapi" "tritonnfs"
 # Needed to add additional programs in the HN's GZ, such as sdc-volapi
 upgrade_gz_tools_to_latest_branch_image tritonnfs
 
-# Now enable the experimental_nfs_shared_volumes flag in SAPI
-sdcadm experimental nfs-volumes
-
+# Now enable the flags in SAPI to enable NFS volumes support for docker and
+# CloudAPI, as well as the ability for docker containers to automount NFS
+# volumes
+sdcadm experimental nfs-volumes docker
+sdcadm experimental nfs-volumes docker-automount
+sdcadm experimental nfs-volumes cloudapi
 EOS
 
 echo "VOLAPI setup done, reboot of all CNs is needed before it can be used"
\ No newline at end of file
-- 
2.21.0

